app/src/main/res/values-fr-rFR/strings.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">CoronaGame</string>
+    <string name="can_t_Download_Offline">Téléchargement hors ligne impossible</string>
+    <string name="can_t_Upload_Offline">Transfert hors ligne impossible</string>
+    <string name="launch_history">Lancer l\'historique</string>
+    <string name="launch_gps">Lancer le GPS</string>
+    <string name="launch_firebase">Lancer Firebase</string>
+    <string name="i_am_infected">Je suis infecté(e)</string>
+    <string name="i_am_cured">Je ne suis pas infecté(e)</string>
+    <string name="gps_status_on">"Signal GPS acquis "</string>
```

Comment:
the double quotes here might be a typo

Reply:
That's a bit surprising indeed, I actually didn't edit this file directly but used the Translations Editor in Android studio. Seems like the quotes don't make a difference